src/loader.c

```diff
@@ -1953,7 +1959,12 @@ static void ldr_show_pw_line(struct db_main *db, char *line)
 
 	if (found && show) {
 		if (source[0])
+		{
 			printf("%c%s", db->options->field_sep_char, source);
+			/* with change to fgetll() vs fgets() source may no longer be \n term */
```

Comment:
Isn't it a latent bug in fgetll() that should be fixed, instead of worked around here?